### PR TITLE
make the invoice page responsive

### PR DESCRIPTION
**Pull request by AdityaDabral**
Targets #2943 

### Description
Added a row in the invoice page and made some additional changes like changing the size of buttons to make the page responsive.

### Checklist:
- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/Invoice/Resources/views/index-filters.blade.php [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <form action="{{ route('invoice.index')  }}" id="invoiceFilterForm">
     <input type="hidden" name="invoice_status" value="{{ request()->input('invoice_status', 'sent') }}">
+
```
Comment (Hemrajbharti123) [inline Modules/Invoice/Resources/views/index-filters.blade.php:3]: remove this unnecessary space

**Review by Hemrajbharti123 — DISMISSED** [review]

**Comment by GauravGusain98** [comment]
@AdityaDabral The buttons and filters UI for small screen is not looking good. Maybe you can try to utilize the blank space and instead of showing filters options vertically put 2 filters in a row. 

Also button text can be reduced so that it comes in one line and the buttons can be shifted to right side
image

**Comment by rahulpanwar16** [comment]
Looks good now.

**Review by rahulpanwar16 — DISMISSED** [review]

**Comment by GauravGusain98** [comment]
### Testing feedbacks
The table is going out of the page. Make it scrollable so that it stays within the page
image

**Review by GauravGusain98 — APPROVED** [review]
Not everything is made responsive but approving as we can go with the pieces that are fixed

**Pull request merged**